CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## v1.17.0 (09/10/2023)
+
+**Common**
+* Improved support of HTTP proxy on iOS.
+* Cleaned up DNS cache on error in cURL (Android).
+* Fix the olp-cpp-sdk-core/CMakeLists.txt to include the externals include directory.
+* Added the possibility to specify DNS using `WithDNSServers` to `olp::http::NetworkSettings` API.
+* Improved logging in `PendingUrlRequests`.
+* Added the `CallApiStream` to `olp::client::OlpClient` API.
```

Comment:
When you write `the SOMETHING` you should specify what that something is. In this case you'd say 
`the CallApiStream THING`. 

If you don't want to call out what `CallApiStream` is, remove "the".